src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -115,8 +120,29 @@ public void setConverter(Converter<Object, byte[]> converter) {
 		setMessageConverter(new ConvertingFromMessageConverter(converter));
 	}
 
+	/**
+	 * When in KPL mode, the setting allows handling backpressure on the KPL native process. Setting this value would enable a sleep on the KPL Thread for the specified number of milliseconds defined in maxRecordInFlightsSleepDurationInMillis.
+	 *
+	 * @param maxRecordsInFlight Defaulted to 0. Value of 0 indicates that Backpressure handling is not enabled. Specify a positive value to enable back pressure.
+	 */
+	public void setMaxOutstandingRecordsInFlight(long maxRecordsInFlight) {
```

Comment:
Name it like `setMaxInFlightRecords`.
The method JavaDoc must come without blank lines.
We also prefer length of code lines not more than `120` symbols.
Also: `@since 3.0.9`

Reply:
fixed.